src/generic/grid.cpp

```diff
@@ -3859,11 +3862,75 @@ void wxGrid::PrepareDCFor(wxDC &dc, wxGridWindow *gridWindow)
     dc.SetDeviceOrigin(dcOrigin.x, dcOrigin.y);
 }
 
+void CheckDoDragScroll(wxGrid *grid, wxGridSubwindow *eventGridWindow,
+    wxGridSubwindow *gridWindow, wxPoint event_pos,
+    bool horizontal, bool vertical)
```

Comment:
Using [bool parameters is a bad idea](https://www.wxwidgets.org/develop/coding-guidelines/#no_bool_params), so
```suggestion
    wxOrientation scrollDirection)
```